File: Python/1074 - Even or Odd.py
N = int(input())
l1 = []
for i in range(N):
    l1.append(int(input()))
for i in range(len(l1)):
    if (l1[i] == 0):
        print("NULL")
    else:
        s = ""
        if (l1[i]%2 == 0):
            s += "EVEN "
        else:
            s += "ODD "
        if (l1[i] > 0):
            s += "POSITIVE"
        else:
            s += "NEGATIVE"
        print(s)

File: C++/1074 - Even or Odd.cpp
#include <iostream>
#include <vector>

std::vector<int> build_vector(int n)
{
    int x;
    std::vector<int> vector1;
    for(int i = 0; i < n; i++)
    {
        std::cin >> x;
        vector1.push_back(x);
    }
    return vector1;
}

void print_results(int n, std::vector<int> vector1)
{
    for(int i = 0; i < n; i++)
    {
        if (vector1[i] == 0)
        {
            std::cout << "NULL\n";
        }
        else
        {
            if(vector1[i] % 2 == 0)
            {
                std::cout << "EVEN ";
            }
            else
            {
                std::cout << "ODD ";
            }
            if (vector1[i] > 0)
            {
                std::cout << "POSITIVE\n";
            }
            else
            {
                std::cout << "NEGATIVE\n";
            }
        }
    }
}

int main() {
    int n;
    std::cin >> n;
    std::vector<int> vector1 = build_vector(n);
    print_results(n,vector1);
    return 0;
}